client/src/components/utils/Header/HamburgerButton/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./style.css";
+
+export default class HamburgerButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      toggled: false
+    };
+  }
+
+  toggleHamburger = () => {
+    const { toggled } = this.state;
+    const { toggleMenu } = this.props;
+    this.setState({ toggled: !toggled }, toggleMenu);
+  };
+
+  render() {
+    const { toggled } = this.state;
+    const { className = "" } = this.props;
+    return (
+      <div
+        className={`hamburger-container ${className} ${
```

Comment:
@AbdallahAmmar96 this is where the npm "classnames" module helps a lot.  It lets you rewrite conditional classnames in a simpler way:
`className={classnames('hamburger-container', className, {'hamburger-container-change': toggled})`
^^ So you can have string literal names, variable names (that are automatically ignored if they are null / undefined), and conditional names that are passed in as an object.

Reply:
Here's the link: https://www.npmjs.com/package/classnames